angles_11:
- 0.3856246564469662
- 0.17035611799872957
- -0.4983709657930019
- -1.9351687341991222
- 0.6207331089780741
- 2.240827951764508
- 0.6569145097426214
angles_10:
- 0.20679780552470894
- -0.6894699210427794
- -0.3434299734858202
- -2.90098825648793
- 0.06158442264795303
- 2.171020208350968
- 0.656915323544042
angles_5:
- -0.0362195368116362
- 0.029972245404818548
- -0.2237965387021459
- -2.096742154941224
- -0.788654154025799
- 2.1794993167142294
- 1.215353530059485
angles_4:
- -0.17345625085579783
- -0.10653875193463334
- -0.23231807205720553
- -2.3587646034261724
- 0.5729185696716109
- 2.504847667614619
- -0.30551586050788554
angles_7:
- 0.35173349978424484
- 0.5989599104011268
- 0.12449616440032657
- -1.5734075362225464
- -0.22470154766241704
- 2.8102920035494594
- 1.8171487520229486
angles_6:
- 0.5024872967974099
- 0.41013633315995757
- -0.37066209452083204
- -1.6674920764793324
- -0.7483440058498477
- 2.2567127783651704
- 1.818520070933612
angles_1:
- -0.2695541701611902
- 0.38283861385311996
- -0.08456704679497501
- -1.5722584562243305
- 0.02792799881100654
- 2.533635051647822
- 0.8355819573096102
angles_0:
- -0.22801163716065073
- -0.32918301972171715
- 0.16082305158230295
- -2.2110587472413714
- 0.02915216634092635
- 1.9381455024180287
- 1.0837174339134021
angles_3:
- -0.3004513795296321
- 0.3005922642632535
- 0.18016433235908413
- -1.6711460986221043
- 1.1105389697257442
- 2.2535973115762076
- -0.5480663007397698
angles_2:
- -0.2633072030732506
- 0.3443493178862062
- 0.24212247803545836
- -1.571330300527828
- 1.1243187139198556
- 2.198980336170873
- 0.5721667947471142
angles_9:
- -0.1719298702666634
- 0.48482122605848665
- -0.1646979187154719
- -1.8960206596642153
- 0.14826380194076863
- 2.8220196066928485
- 0.5521225164425042
angles_8:
- -0.19681383937910985
- 0.06483176855934844
- -0.38984688791604893
- -2.0196242296653883
- -0.22697117533591252
- 2.3265667948457924
- -0.47325509249253406
